test/e2e/page-objects/pages/home/asset-list.ts

```diff
@@ -0,0 +1,269 @@
+import { Driver } from '../../../webdriver/driver';
+
+class AssetListPage {
+  private readonly driver: Driver;
+
+  private readonly allNetworksOption =
+    '[data-testid="network-filter-all__button"]';
+
+  private readonly allNetworksTotal =
+    '[data-testid="network-filter-all__total"]';
+
+  private readonly assetOptionsButton = '[data-testid="asset-options__button"]';
+
+  private readonly confirmImportTokenButton =
+    '[data-testid="import-tokens-modal-import-button"]';
+
+  private readonly confirmImportTokenMessage = {
+    text: 'Would you like to import this token?',
+    tag: 'p',
+  };
+
+  private readonly currentNetworkOption =
+    '[data-testid="network-filter-current__button"]';
+
+  private readonly currentNetworksTotal = `${this.currentNetworkOption} [data-testid="account-value-and-suffix"]`;
+
+  private readonly hideTokenButton = '[data-testid="asset-options__hide"]';
+
+  private readonly hideTokenConfirmationButton =
+    '[data-testid="hide-token-confirmation__hide"]';
+
+  private readonly hideTokenConfirmationModalTitle = {
```

Comment:
This is similar css and text format we should avoid. As you have already mentioned, you could make a note of this too.

Reply:
Sounds good, thanks !